src/main/webapp/i18n/en/exercise.json

```diff
@@ -181,13 +181,15 @@
             "isTeamExercise": "This is a team exercise.",
             "update": {
                 "warning": {
-                    "warning": "Warning!",
-                    "confirm": "Confirm changes",
-                    "problems": "The following problems may occur:",
-                    "instructionDeleted": "Grading Instruction deleted:",
-                    "scoringChange": "Scoring changed:",
-                    "results": "The existing results must be recalculated!",
-                    "confirmtext": "Do you really want to perform these changes?"
+                    "warning": "Warning: This change will possibly cause problems",
+                    "saveExerciseWithoutReevaluation": "Do not Re-evaluate",
+                    "problems": "To avoid inconsistencies between existing feedback and future feedback that is associated to affected grading instruction, the exercise has to be re-evaluated. This includes the recalculation of the results.",
+                    "instructionDeleted": "Grading instruction deleted",
+                    "scoringChange": "Grading instruction credit changed",
```

Comment:
is credit correct here? isn't it scoring?

Reply:
noticed that score can be confusing in here. cuz, we have `credit` for grading instruction. I will update the `scoringchange` with `creditChange`